### PR TITLE
Add monitoring label

### PR DESCRIPTION
**Pull request by omegas27**

### CODE REVIEW

File: helm/azure-operator/templates/service.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: {{ include "resource.default.namespace"  . }}
   labels:
     {{- include "labels.common" . | nindent 4 }}
+    giantswarm.io/monitoring: "true"
```
Comment (fiunchinho) [inline helm/azure-operator/templates/service.yaml:8]: What do you think about adding it as [a _common_ label here](https://github.com/giantswarm/azure-operator/blob/bb6a7330bbe7bf28bd799218b658252d632468f5/helm/azure-operator/templates/_helpers.tpl#L19)?

Comment (omegas27) [reply helm/azure-operator/templates/service.yaml:8]: If we do, the label will be everywhere when we only need it on the service so we can scrape it using the new prometheus setup

Comment (fiunchinho) [reply helm/azure-operator/templates/service.yaml:8]: Oh you are right. Never mind me then :smiley_cat:

Comment (omegas27) [reply helm/azure-operator/templates/service.yaml:8]: Well, now you know :dog: 
If we actually add it everywhere, we might scrape the endpoints multiple times

**Review by mig4 — APPROVED** [review]

**Pull request merged**